tests/merge_test.py

```diff
@@ -50,9 +52,9 @@ def test_merge_from_fixtures(tmp_path):
     general_merge(base_path, extra_path, None, outpath)
     with open(outpath, "r") as f:
         l = f.readlines()
-        final_network_count = len(l)
-        expected_count = (len(base_nets) + len(extra_nets)) - (base_subnet_count + extra_subnet_count)
-        assert final_network_count == expected_count
+        final_networks = {line.split()[0] for line in l}
+        expected_networks = set(base_nets + extra_nets) - set(base_subnets + extra_subnets)
```

Comment:
Hm, why do we need the `set` here? Doesn't the merge code handle duplicates as well?

Reply:
And if duplicates is not a case we cover here yet, I would suggest you can just add it basically as another variant of the subnets test cases.

Reply:
I think this is more complicated than it has to be. I sketched out below what I was thinking of since that's clearer than describing it :) Let me know if it makes sense.